src/markdown-pages/copilot.md

```diff
@@ -0,0 +1,105 @@
+---
+slug: "articles/copilot"
+date: 2021-06-30
+title: "Copilot and Conversational Programming"
+pageType: article
+---
+
+# Conversational Programming
+A month ago, I got a demo of GitHub's [Copilot](https://copilot.github.com/) and I've been pondering the implications. Here's what I've realized: Copilot won't replace programmers, but it will change what it means to be a programmer. And that's a very good thing.
+
+But before I can introduce the change, we need to talk about what programming is, where we are in the history of programming, and where we're going.
+
+## Programming As Basic Literacy
+Humans are Tool Makers. Programming represents one of the most powerful tools humanity has yet created, on par with the invention of spoken language and writing. Like speaking and writing, programming is an expressive medium. Programming is the ability to express one's will, to take "that which you wish to be true" and manifest it in the world. Speaking enables you to communicate your ideas. Writing enables you to capture and transmit them. Programming enables you to make your ideas executable. Moreover, programming will become a fundamental human capability. Like reading and writing before it, programming will morph and adapt over the next few hundred years and eventually spread through the human race until it's considered part of "basic literacy."
```

Comment:
"Moreover, programming will become a fundamental human capability." This feels like a bold claim that could use more supporting evidence.

Reply:
@nickbasile I desperately want to respond to this objection, as it's the part of the article I'm perhaps most interested in. But upon further reflection, I realize that this whole section of the article is a distraction.

My main thesis is we're moving from dictation to conversation. Everything should be driving towards that.

I'm going to cut the section on scribes (but save for another article!), tighten up the latter sections, and see if the article "works" better.